Armadillo/src/main/java/com/scribd/armadillo/error/ArmadilloException.kt

```diff
@@ -79,6 +79,16 @@ object UnableToLoadDownloadInfo : ArmadilloException(Exception()) {
     override val errorCode = 303
 }
 
+/**
+ * Occurs when a device supporting Android 12 (SDK 31+) attempts to launch the ExoPlayer DownloadService while the user has put the app
+ * into the background, often during low connectivity when the DownloadHelper.prepare() stalls long enough for the user to put the app in
```

Comment:
I wonder if this means we need to move `DownloadHelper.prepare` into the service?

Reply:
`DownloadHelper.prepareI()` is the step necessary to launch the audiobook download service with `DownloadService.sendAddDownload()`

Our `DownloadService` is only for ebooks